README.md

```diff
@@ -178,4 +208,49 @@ so for instance...
+
+- You can add dev-tools to the array of services and troubleshoot docker using portainer (mentioned above). Make sure that you have added also the URL to the hostfile (`/etc/hosts`) and that it matches the variable defined in `.env`
+
+- If you remove services from array `ENABLED_SERVICES`, they are not stopped. This is not a problem in general, just don't be surprised if when you start (having the configuration `ENABLED_SERVICES=core:frontend:events`) and stop later (with the configuration `ENABLED_SERVICES=events`) you find a message `WARNING: found orphan containers`
+
+## How to run unit/integration tests when you start the system
+
+?? @serge1peshcoff
+
+## Default credentials
+
+From the [core readme](./core/README.md): there are test users. All users have `5ecr3t5ecr3t` as password.
+
+- `admin@example.com` - admin
+- `board@example.com` - board member of antenna
+- `member@example.com` - regular member of antenna
+- `not-confirmed@example.com` - member who is not confirmed
+- `password-reset@example.com` - member who requested a password reset
+- `suspended@example.com` - a suspended member
+
+You can use `5ecr3t` for a password reset token (for a member with email `password-reset@example.com`) and `5ecr3t` for a mail confirmation (for a member with email `not-confirmed@example.com`).
+
+**NOTICE** if you use the `--fast` mode of `start.sh`, then core does NOT provision in production mode, so you will be fast in bootstrapping but without users to play with.
+
+@serge1peshcoff please confirm
+
+## How to reset the database and recreate
+
+A couple of options
+
+1) As mentioned above, you can enable dev-tools for pgadmin. From there you can delete the db
+  - Notice unfortunately you have to configure pgadmin when you first login: specify the host, username, password.
+    - Username and password of the database is NOT the same username and password of pgadmin. Find everything in the files mentioned in 'moving parts', or `current-config.yml` for ease.
+    - You put the name of the service as hostname (docker internally resolves stuff with its internal DNS). In other words, every container is reachable at the host named like its service (e.g. `frontend`, `core`, `postgres-core`, etc). The 'service' is named in the `docker-compose.yml` under the key `services:`. Again: _the title of the container is also the hostname of the container_
+
+2) Use portainer to delete the service and db, then `make start` to fix everything.
+  - The service must be deleted/restarted because it runs the migrations and therefore fills the DB with the important data. If you delete only the db, the service will expect the db to be filled with data, causing errors
```

Comment:
I'll enhance this with a few words on how to run a manual `SELECT * from table` with `pgadmin`. I'll add an example for the steps we did today together too.

Reply:
do we have pgadmin working though?